views/partials/subscribe/mailchimp.hbs

```diff
@@ -0,0 +1,73 @@
+<div class="container">
+  <div class="col-8 offset-2">
+    <!-- Begin MailChimp Signup Form -->
+    <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
+    <style type="text/css">
+      #mc_embed_signup {
+        background: #fff;
+        clear: left;
+        font: 14px Helvetica, Arial, sans-serif;
+      }
+
+      /* Add your own MailChimp form style overrides in your site stylesheet or in this style block.
+	   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
+    </style>
+    <div id="mc_embed_signup">
+      <form action="https://die-lernwerkstatt.us10.list-manage.com/subscribe/post?u=63598878eb7adee616350bca7&amp;id=9367cfd8d5"
+        method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank"
+        novalidate="novalidate">
+        <div id="mc_embed_signup_scroll">
+          <h2 class="text-center">Hier kannst du dich in unseren Newsletter eintragen</h2>
+          <div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
+          <div class="mc-field-group">
+            <label for="mce-EMAIL">E-Mail-Adresse <span class="asterisk">*</span>
+            </label>
+            <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" aria-required="true">
+          </div>
+          <div class="mc-field-group">
+            <label for="mce-FNAME">Vorname </label>
+            <input type="text" value="" name="FNAME" class="" id="mce-FNAME">
+          </div>
+          <div class="mc-field-group">
+            <label for="mce-LNAME">Nachname </label>
+            <input type="text" value="" name="LNAME" class="" id="mce-LNAME">
+          </div>
+          <p><a href="https://us10.campaign-archive.com/home/?u=63598878eb7adee616350bca7&amp;id=9367cfd8d5" title="View previous campaigns">View
+              previous campaigns.</a></p>
+          <div id="mce-responses" class="clear">
+            <div class="response" id="mce-error-response" style="display:none"></div>
+            <div class="response" id="mce-success-response" style="display:none"></div>
+          </div>
+          <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+          <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_63598878eb7adee616350bca7_9367cfd8d5"
+              tabindex="-1" value=""></div>
+          <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+        </div>
+      </form>
+    </div>
+    <script type="text/javascript" src="//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js"></script>
```

Comment:
Let's leave it for now, but later, I think we can get rid of this validator in favor of standard HTML5.